src/components/RuntimeManager.js

```diff
@@ -81,14 +81,14 @@ export const RuntimeErrorModal = ({ runtime, onDismiss }) => {
           .then(res => res.text()))
       setUserscriptError(true)
     } else {
-      setError(runtimeErrors[0].errorMessage)
+      setError(runtimeErrors[0]?.errorMessage)
```

Comment:
Since it looks like `runtimeErrors` may be empty, does this need a default error?

Reply:
This shouldn't happen in the wild, I was only running into it because I was manually modifying my runtime in the database... It can't hurt though may as well

Reply:
For what it's worth, if it shouldn't happen in the wild, I think the former `setError(runtimeErrors[0].errorMessage)` would be fine. Adding the `?` operator made it seem like an empty array was a case that was expected and needed to be handled.